GUI/DeusEx/Classes/DXRMenuScreenNewGame.uc

```diff
@@ -73,15 +73,19 @@ event InitWindow()
     StyleChanged();
 }
 
-
 function SetDxr(DXRando d)
 {
     dxr=d;
     flags = DXRFlags(dxr.FindModule(class'DXRFlags'));
-    if(flags.moresettings.splits_overlay > 0)
+
+    if(
+        class'MenuChoice_ShowNewSeed'.default.value == 1 ||
+        (class'MenuChoice_ShowNewSeed'.default.value == 0 && flags.moresettings.splits_overlay > 0)
```

Comment:
we could move this logic into `MenuChoice_ShowNewSeed` with a static function like we do for the others